spec/integration/build-swift/after/Example/Example.xcodeproj.yaml

```diff
@@ -0,0 +1,168 @@
+
+File References:
```

Comment:
Have a look at [spec/spec_helper/xcodeproj_project_yaml](https://github.com/mrackwitz/xcres/blob/master/spec/spec_helper/xcodeproj_project_yaml.rb).

Unfortunately this is a one way street. Great for checking the changes in an xcodeproj. But you can't actually feed Xcode with YAML files.